src/controllers/voteController.js

```diff
@@ -0,0 +1,100 @@
+// express async handler
+const asyncHandler = require("express-async-handler");
+
+// vote model
+const Vote = require("../models/voteModel");
+
+// POST /vote/
+const addVote = asyncHandler(async (req, res) => {
+  const { body } = req;
+
+  const add_vote = new Vote(body);
+
+  try {
+    // check if action is valid
+    if (body.action !== "VOTE_UP" && body.action !== "VOTE_DOWN") {
```

Comment:
the enum ensures the option is either `VOTE_UP` or `VOTE_DOWN`, no need to validate it

Reply:
- [ x ]

Reply:
Done